shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -186,8 +180,14 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
    * @param replication The replication of file to load into Alluxio memory
    */
   private JobAttempt newJob(AlluxioURI filePath, int replication) {
+    final ClientContext clientContext = ClientContext.create(mFsContext.getPathConf(filePath));
```

Comment:
Do you need the path-conf here?

Reply:
I left it currently as the first context of the first file it hits though it may not be necessary.

Reply:
If job_worker is the one that eventually uses the path-conf, then yeah.